### PR TITLE
Try testing splitting reporting workflow"

### PR DESCRIPTION
**Pull request by CBonade**
## Original issue(s)
department-of-veterans-affairs/va.gov-team#0000

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: .github/workflows/testing-reports.yml [hunk 1]
```diff
@@ -23,3 +23,114 @@ jobs:
...
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/testing-tools-team-dashboard-data
+          token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
+          path: testing-tools-team-dashboard-data
+
+      - name: Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14 # ${{ steps.get-node-version.outputs.NODE_VERSION }}
```
Comment (oseasmoran73) [inline .github/workflows/testing-reports.yml:40]: We can prob remove comment?

File: .github/workflows/testing-reports.yml [hunk 2]
```diff
@@ -23,3 +23,114 @@ jobs:
...
+        env:
+          YARN_CACHE_FOLDER: .cache/yarn
+        working-directory: testing-tools-team-dashboard-data
+
+      - name: Download Mochawesome test results
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          github_token: ${{secrets.GITHUB_TOKEN}}
+          workflow: continuous-integration.yml
+          workflow_conclusion: success
```
Comment (oseasmoran73) [inline .github/workflows/testing-reports.yml:53]: Would we want it on `success`? I thought we would still want to download artifacts even fail (cypress/nightwatch)

File: .github/workflows/testing-reports.yml [hunk 3]
```diff
@@ -23,3 +23,114 @@ jobs:
...
+
+      - name: Download Mochawesome report artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: mochawesome-report
+          path: testing-reports/cypress-reports
+
+      - name: Download video artifacts
+        if: ${{ needs.cypress-tests.result == 'failure' }}
+        uses: actions/download-artifact@v2
```
Comment (oseasmoran73) [inline .github/workflows/testing-reports.yml:104]: Would this need to use `dawidd6/action-download-artifact@v2`?

**Review by oseasmoran73 — APPROVED** [review]

File: .github/workflows/testing-reports.yml (hunk 2 above)
Comment (CBonade) [reply .github/workflows/testing-reports.yml:53]: Good call, removed this line

File: .github/workflows/testing-reports.yml (hunk 1 above)
Comment (CBonade) [reply .github/workflows/testing-reports.yml:40]: removed

**Review by oseasmoran73 — APPROVED** [review]

**Pull request merged**